annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodDeclarationTree.java

```diff
@@ -46,7 +48,9 @@ public class MethodDeclarationTree extends MetaData<MethodNode> {
   private HashMap<Integer, MethodNode> nodes;
 
   /** Set of all classes flat name declared in module. */
```

Comment:
Now we have two identical comments (here and line 52).  Should one be updated?

Reply:
Sorry for that, fixed it in 4ad8230a3ca6e70aa3fe9531e7844d590ec68068